include/tvm/ir/module.h

```diff
@@ -349,6 +390,9 @@ class IRModuleNode : public Object {
    */
   std::unordered_set<String> import_set_;
   friend class IRModule;
+
+ public:
+  void ExtractPrimFuncConstants(BaseFunc func);
```

Comment:
Shall we rename this to  "ExtractConstants" ?

Reply:
Renamed